src/hockeypuck/hkp/sks/recon.go

```diff
@@ -83,7 +84,7 @@ func NewPrefixTree(path string, s *recon.Settings) (recon.PrefixTree, error) {
 	return leveldb.New(s.PTreeConfig, path)
 }
 
-func NewPeer(st storage.Storage, path string, s *recon.Settings, opts []openpgp.KeyReaderOption) (*Peer, error) {
+func NewPeer(st storage.Storage, path string, s *recon.Settings, opts []openpgp.KeyReaderOption, userAgent string) (*Peer, error) {
```

Comment:
At some point, we might want to refactor this constructor to take a `PeerOptions` struct with all these different options in it.

Reply:
Sounds like a good improvement indeed else cognitive complexity will keep to grow.

P.S. Give me a few to remove the version bump of golang in `debian/control` that went in by accident. Or probably we keep it as golang 1.13 is what comes with latest Ubuntu 20.04? :thinking: